migtests/tests/oracle/datatypes/validate

```diff
@@ -81,5 +88,18 @@ def migration_completed_checks(tgt):
 			print(f"column_name: {column_name}, datatype: {datatype}")
 			assert datatype == EXPECTED_DATA_TYPES[table_name][column_name]
 
+	fetched_value_v1_column_in_varray_table = tgt.execute_query("SELECT v1 FROM varray_table where id = 10")
+	print(fetched_value_v1_column_in_varray_table)
+	expected_value = '("{(cde,fgh,ghi,jkl)}")'
+	if not os.environ.get("USE_DEBEZIUM") == '1':
+		assert fetched_value_v1_column_in_varray_table == expected_value
+	else:
+		assert fetched_value_v1_column_in_varray_table == None
+
+	fetched_value_rid_column_in_varray_table = tgt.execute_query("SELECT rid FROM varray_table where id = 9")
+	print(fetched_value_rid_column_in_varray_table)
+	assert fetched_value_rid_column_in_varray_table == None
```

Comment:
Why assert None here?

Reply:
its ROWID type column, so after ignoring nothing will be imported.

Reply:
Got it. Pls add a comment here explaining

Reply:
just added in latest commit.